bin/alluxio-start.sh

```diff
@@ -52,8 +52,7 @@ check_mount_mode() {
     Mount);;
     SudoMount);;
     NoMount)
-      mount | grep "${ALLUXIO_RAM_FOLDER}" > /dev/null
-      if [[ $? -ne 0 || -z ${ALLUXIO_RAM_FOLDER} ]]; then
+      if [[ $(mount | grep -c ${ALLUXIO_RAM_FOLDER}) != 0 ]]; then
```

Comment:
I am confused here. I intended to override ALLUXIO_RAM_FOLDER if user specifies "NoMount" and $ALLUXIO_RAM_FOLDER is not mounted. 

For example:
./bin/alluxio-start.sh local NoMount => use /dev/shm
If the user manually mounted /mnt/ramdisk, then ./bin/alluxio-start.sh local NoMount => use /mnt/ramdisk

I just tried it, it seems working as expected. Maybe I misunderstood something here.